mbuild/utils/conversion.py

```diff
@@ -21,3 +21,325 @@ def RB_to_OPLS(c0, c1, c2, c3, c4, c5):
     f3 = -0.5 * c3
     f4 = -0.25 * c4
     return np.array([f1, f2, f3, f4])
+
+
+def RB_to_CHARMM(c0, c1, c2, c3, c4, c5):
```

Comment:
All of these helper function should be super easy to cover with tests.

Reply:
Yeah.  I think I have all these covered.  Mostly, the only bad spots are the places where errors are hard to generate, or I need to make additional FF files or other things to get an error.